src/pages/myNotes/myNotesDesctiption.jsx

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+
+const MyNoteDesctiption = ({ activeNote }) => {
+  if (!activeNote) return <div className="no-active-note"  style={{ flex: "3" }}>No Active Note</div>;
```

Comment:
Please, try not to use inline styles, describe all of them inside styles.js file